Source/WebKit/Platform/spi/Cocoa/ExtensionKitSPI.h

```diff
@@ -41,6 +41,8 @@ typedef void(^_SEServiceInteruptionHandler)();
 
 @interface _SECapabilities : NSObject
 +(instancetype)assertionWithDomain:(NSString*)domain name:(NSString*)name;
```

Comment:
Do we still need to overload without the `environmentIdentifier` argument? I guess it doesn't hurt to leave it, but if we don't need it it's a bit tidier to remove it.

Reply:
Ah, that is a good point. I will address it in a follow-up patch.

Thanks for reviewing!